testgrid/config.yaml

```diff
@@ -6754,6 +6759,10 @@ dashboards:
     test_group_name: pull-charts-e2e
   - name: cadvisor
     test_group_name: pull-cadvisor-e2e
+  - name: gcp-k8s-cluster-bundle
+    test_group_name: pull-k8s-cluster-bundle-bazel-test
+  - name: gcp-k8s-cluster-bundle-post
```

Comment:
I don't think we can expect users to know that this maps to `post-k8s-cluster-bundle-bazel-test`. Is there motivation for not using the job names (or a shortened version of the name) here and above?

Reply:
Not really, changed.